lib/matplotlib/tests/test_dates.py

```diff
@@ -1226,3 +1232,58 @@ def test_julian2num():
     mdates.set_epoch('1970-01-01T00:00:00')
     assert mdates.julian2num(2440588.5) == 1.0
     assert mdates.num2julian(2.0) == 2440589.5
+
+
+def test_rcparams_timezone():
+    assert mdates._get_rc_timezone() == mdates.UTC
```

Comment:
It's generally better to test public facing API instead of private API - can this be modified to hit the `_get_rc_timezone()` function with a public API call?